package.json

```diff
@@ -23,8 +23,11 @@
   },
   "homepage": "https://github.com/founders-programme-2/business-launchpad#readme",
   "dependencies": {
+    "bcrypt": "^3.0.6",
     "concurrently": "^4.1.0",
-    "express": "^4.17.1"
+    "dotenv": "^8.0.0",
```

Comment:
what the different between env2 & dotenv

Reply:
I think they are basically the same, but dotenv is more stable as it as more weekly downloads and now that Dwyl is closed, I am not sure who will be maintaining their packages.

Here are the docs for dotenv
https://npm.taobao.org/package/dotenv

It "loads environment variables from a .env file into process.env"